tool/site/client-react/src/components/prescription-tracker/prescriptionSearch.tsx

```diff
@@ -28,7 +49,7 @@ const PrescriptionSearch: React.FC<PrescriptionSearchProps> = ({
   return (
     <>
```

Comment:
can remove fragment here

Reply:
doesn't compile for me when i attempted to remove